js/ui/components/codingGameService.js

```diff
@@ -1,52 +1,152 @@
 // -*- mode: js; js-indent-level: 4; indent-tabs-mode: nil -*-
 
 const Gio = imports.gi.Gio;
+const GLib = imports.gi.GLib;
 const Lang = imports.lang;
 const Meta = imports.gi.Meta;
-const Showmehow = imports.gi.Showmehow;
+const CodingGameService = imports.gi.CodingGameService;
 
 const CodingChatboxTextService = new Lang.Class({
     Name: 'CodingChatboxTextService',
 
     enable: function() {
         // Connect to the service
         try {
-            this._service = Showmehow.ServiceProxy.new_for_bus_sync(
+            this._service = CodingGameService.CodingGameServiceProxy.new_for_bus_sync(
                 Gio.BusType.SESSION, 0,
-                'com.endlessm.Showmehow.Service',
-                '/com/endlessm/Showmehow/Service', null);
+                'com.endlessm.CodingGameService.Service',
```

Comment:
There is an issue with the naming in the service file. This will fix it for the autostart, or you change it here.

```
diff --git a/com.endlessm.CodingGameService.service.in b/com.endlessm.CodingGameService.service.in
index 6dde9a7..4893f23 100644
--- a/com.endlessm.CodingGameService.service.in
+++ b/com.endlessm.CodingGameService.service.in
@@ -1,3 +1,3 @@
 [D-BUS Service]
-Name=com.endlessm.CodingGameService
+Name=com.endlessm.CodingGameService.Service
 Exec=@bindir@/coding-game-service
```